Readme.md

```diff
@@ -7,9 +7,13 @@ Mod for Factorio. *Store **all** the things!* (We heard you like boxes, you pack
 
 [![GitHub release](https://img.shields.io/github/release/Anoyomouse/Warehousing.svg?label=current+version)]()
 [![GitHub Release Date](https://img.shields.io/github/release-date/Anoyomouse/Warehousing.svg?label=released)]()
+[![Crowdin](https://d322cqt584bo4o.cloudfront.net/factorio-warehousing/localized.svg)](https://crowdin.com/project/factorio-warehousing)
 
 Go grab it now, or get it in game by searching for Warehousing.
 
+### Translations!
+We have been approved as an open source crowdin project, please help us translate the mod on their site (needs a free account, or just use google, twitter, github, or others), join in and help us make this mod awesome
```

Comment:
What do you think of this?

> Want to see Warehousing available in your language? Does the quality of our existing translation leave you wanting more? Please help us [translate the mod](https://crowdin.com/project/factorio-warehousing) on Crowdin! Create a free account (or log in with Google, Twitter, Facbook, GitHub, or Gitlab) and help make this mod (even more) awesome!

Too much? Don't want to possibly insult the previous locale contributors by implying that some of their work isn't perfect? :grinning: I'm not sure we need to get into Crowdin's supported third-party services for login (I just made the list because you included it), but forewarning that an account is required is nice.

Reply:
Noting for the record that my edit was :+1:'d in Steam chat.

````
<anoyomouse-st>  That looks good
<anoyomouse-st>  (answering your latest github PR here) because my PC is being an ass >.>
<dgw>  Your PC is being an ass in such a way that you can't post on github? That's new
<anoyomouse-st>  oh, it's just slow, at least chrome is .. but steam is fine :P
<dgw>  HA
<dgw>  usually I find it's the other way around
<dgw>  You said my latest PR, but I assume that actually means the comment/review on the Crowdin readme PR you started
<anoyomouse-st>  yes, the latest thing on #40 :P
````